src/Sylius/Bundle/ShopBundle/Tests/Security/ShopFirewallTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Tests\Security;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class ShopFirewallTest extends WebTestCase
```

Comment:
ShopFirewallTest should be final because Tests should not be extendable

```suggestion
final class ShopFirewallTest extends WebTestCase
```